src/System.Data.SqlClient/src/System/Data/SqlClient/SqlException.cs

```diff
@@ -11,25 +11,47 @@
 using System.Data.Common;
 using System.Diagnostics;
 using System.Globalization;
+using System.Runtime.Serialization;
 using System.Text; // StringBuilder
 
 namespace System.Data.SqlClient
 {
+    [Serializable]
     public sealed class SqlException : System.Data.Common.DbException
```

Comment:
We can add it there later if we want, but that won't work until it consumes this package with the serialization attribute.  That test also wasn't meant to be an everlasting, all-encompassing test of everything serializable in corefx; rather, it was meant to help test BinaryFormatter in the BinaryFormatter test suite on a bunch of different serialization inputs (especially since it builds against packages and thus we want to be able to catch regressions in BinaryFormatter as part of checking in changes to BinaryFormatter rather than waiting until it gets merged and into a package).